### DISCUSSION TITLE
Render `org-mode` in the app

### DISCUSSION THREAD

**Discussion by cole-jacobs** [community/community#4944] [Mobile]
Org-mode is rendered in the browser version of this site, but in the iOS app I only see the plain-text versions of my files. I searched through this feedback repo, and it looks like I'm the only one mentioning this... so maybe the demand isn't there. I think it would be awesome though (and hopefully not very difficult seeing as there is already a parser being used in the browser).

**Comment by eliperkins** [comment]
Thanks for this feedback, @cole-jacobs! TIL that GitHub.com supports rendering Org-mode 😅 

I've added your notes to an internal issue to track this. I'll let you know when this gets addressed!
